src/sign.ts

```diff
@@ -20,11 +20,12 @@ import {
   SigstoreBlobBundle,
```

Comment:
Would be good to add a test case to the signBlob and signAttestation expecting the leaf cert to be sent to rekor

Reply:
Good idea. I updated one of the tests to check this.